okhttp/src/main/java/okhttp3/CertificatePinner.kt

```diff
@@ -275,7 +295,7 @@ class CertificatePinner internal constructor(
       }
     }
 
-    override fun toString(): String = hashAlgorithm + hash.base64()
+    override fun toString(): String = hashAlgorithm + "/" + hash.base64()
```

Comment:
This seems like the better external api, with the slash removed

Reply:
Definitely.

Reply:
Template string?